aces/idt/core/common.py

```diff
@@ -756,3 +758,153 @@ def format_exposure_key(key: str) -> str:
         return f"+{key_float:g}"
     else:
         return f"{key_float:g}"
+
+
+def get_clipping_threshold() -> float:
+    """Get the threshold for determining if RGB values are clipped. The threshold is
+        2 code values in a 10 bit system
+
+    Returns
+    -------
+    float
+        The threshold for determining if RGB values are clipped
+
+    """
+    bit_depth = 10
+    code_value_threshold = 2
+    max_code_value = 2**bit_depth - 1
+    threshold = float(code_value_threshold) / max_code_value
+    return threshold
+
+
+def find_close_indices(data: np.array, threshold: float = 0.005) -> list:
+    """
+    Find the indices of rows that have any values with differences below the threshold.
+
+    Parameters
+    ----------
+    data : np.array
+        A numpy array of shape (n, 3, 3) where each row contains RGB values.
+    threshold : float, optional
+        The tolerance threshold for determining if the RGB values between consecutive
+        rows are similar. Default is 0.005.
+
+    Returns
+    -------
+    List[int]
+        A list of indices where the rows have RGB differences below the threshold.
+    """
+    top_indices = []
+    bottom_indices = []
+    n_rows = data.shape[0]
+
+    # Search from the top
+    for i in range(n_rows - 1):
+        if np.any(np.abs(data[i] - data[i + 1]) < threshold):
+            top_indices.append(i)
+        else:
+            break
+
+    # Search from the bottom
+    for i in range(n_rows - 1, 0, -1):
+        if np.any(np.abs(data[i] - data[i - 1]) < threshold):
+            bottom_indices.append(i)
+        else:
+            break
+
+    # Combine and return the indices
+    return sorted(np.array(top_indices + bottom_indices))
+
+
+class ExposureClippingFilter:
```

Comment:
Do we really need a class for this? Everything is done in the `ExposureClippingFilter.filter_samples` method and the objects are never ever changed, seems like heavy handed.

Reply:
Yeah agree, i was thinking if we ever needed to overload any of this but i think your right, can probably just be a function